minimint-api/Cargo.toml

```diff
@@ -24,6 +24,7 @@ thiserror = "1.0.23"
 tracing ="0.1.22"
 
 [target.'cfg(target_family="wasm")'.dependencies]
+async-lock = "2.5"
```

Comment:
So is `async-lock` only for wasm, otherwise use tokio?

Reply:
yes